stories/s2l2a.stories.js

```diff
@@ -843,6 +843,93 @@ export const getMapProcessingGainGammaCheckTransparency = () => {
   return wrapperEl;
 };
 
+export const getMapProcessingAdvancedRGB = () => {
+  const imgOriginal = document.createElement('img');
+  imgOriginal.width = '256';
+  imgOriginal.height = '256';
+
+  const imgRGB1 = document.createElement('img');
+  imgRGB1.width = '256';
+  imgRGB1.height = '256';
+
+  const imgRGB2 = document.createElement('img');
+  imgRGB2.width = '256';
+  imgRGB2.height = '256';
+
+  const wrapperEl = document.createElement('div');
+  wrapperEl.innerHTML = '<h2>S2L2A getMapProcessingAdvancedRGB</h2>';
+  wrapperEl.innerHTML += '<h4>no effects | same value | function</h4>';
```

Comment:
Nitpick, FFTR: I would suggest using a bit different naming:
```
<h4>original | no change | black & white</h4>
```
(assuming you use the suggested function for the last effect)

Reply:
Thanks, changed (a bit better than the existing).